src/Microsoft.ML.Transforms/GcnTransform.cs

```diff
@@ -25,7 +25,7 @@
 
 [assembly: EntryPointModule(typeof(LpNormalization))]
 
-namespace Microsoft.ML.Runtime.Data
+namespace Microsoft.ML.Transforms.Projections
```

Comment:
>Projections [](start = 34, length = 11)

PCA, Whitening, LDA at some extent is also projections.

Reply:
is LDA projections or text?

Reply:
it's projection of text! so Text, (Well it's operate on vector of floats which we get from word bag, but it's main purpose is text related)
I forgot RFF, it's also projection.
Actually Whitening on internal side is stated in Image category. 

---
In reply to: [229512989](https://github.com/dotnet/machinelearning/pull/1453#discussion_r229512989) [](ancestors = 229512989)

Reply:
Well, fun fact, GCN is also stated as Image transform :)

---
In reply to: [229514079](https://github.com/dotnet/machinelearning/pull/1453#discussion_r229514079) [](ancestors = 229514079,229512989)